tests/run-make/rustdoc-scrape-examples-multiple/src/lib.rs

```diff
@@ -1,6 +1,5 @@
-// @has foobar/fn.ok.html '//*[@class="docblock scraped-example-list"]//*[@class="prev"]' ''
```

Comment:
@GuillaumeGomez IIUC, this specific directive, checking for the a prev button, was broken by #129796 , and is no longer needed?

Reply:
The prev button is generated by the JS, so it cannot be checked here indeed.